src/core/io_queue.cc

```diff
@@ -297,6 +298,10 @@ class queued_io_request final : private internal::io_request, public fair_queue_
     queued_io_request(queued_io_request&&) = delete;
     ~queued_io_request() = default;
 
+    virtual throttle::grab_result can_dispatch() const noexcept {
```

Comment:
It seems that the compiler on CI complains that it does not use `override` keyword.

```
io_queue.cc:301:35: error: 'can_dispatch' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
  301 |     virtual throttle::grab_result can_dispatch() const noexcept {
```